content/swiftui/concepts/views/terms/toggle/toggle.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'Toggle'
+Description: 'Chooses between an "on or off" state based on a Bool type value.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Toggle'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **`Toggle`** is a View that chooses between "on or off" options based on a `Bool` type value.
+
+## Syntax
+
+```pseudo
+Toggle("Title String", isOn: Binding<Bool>)
+```
+
+The following parameters can be applied to a `Toggle` view:
+
+- The `"Title String"` parameter is used instead of a [closure](https://www.codecademy.com/resources/docs/swift/closures).
```

Comment:
Can we also include that this title parameter is optional? (The `_` doesn't indicate the parameter is optional, but it is an optional param 🙂 )